e2e/internal/e2e/singularitycmd.go

```diff
@@ -72,6 +72,10 @@ const (
 )
 
 func (r *SingularityCmdResult) expectMatch(mt MatchType, stream streamType, pattern string) error {
+	if pattern == "" {
```

Comment:
I'm not sure why this was not there before. Is there a requirements for a command to have no output?

Reply:
A case I think of is to test if Singularity correctly didn't show a warning/info message for a command where you could use something like `e2e.ExpectExit(0, e2e.ExpectError(e2e.ExactMatch, ""))`